manifests/install_publish.pp

```diff
@@ -76,7 +76,7 @@
     puppet_conf_dir         => '/etc/puppetlabs/puppet/',
     data_volume_device      => $data_volume_device,
     data_volume_mount_point => $data_volume_mount_point,
-    run_mode                => $run_mode,
+    run_modes               => $run_modes,
```

Comment:
`install_publish` needs to enforce `publish` to be added to user `run_modes` here.
If we simply pass `run_modes` as-is, then there's no point of calling this class `install_publish` .

Alternatively, set `run_modes` param to default to `['publish']`